### ISSUE TITLE
Parser doesn't recognise some functions from Dear ImGui library

### ISSUE THREAD

**Issue by yyamdev** [slynch8/10x#408]
(Seen in version 0.28.27 on 64 bit Windows)

Just a super tiny thing I noticed: The Text... functions in [Dear ImGui](https://github.com/ocornut/imgui) are not recognised by the parser:
![dear imgui](https://user-images.githubusercontent.com/9505900/132763860-703b69ad-61d5-4b0f-af37-568a302cb3d2.png)

Probably because of the weird macro straight after them!

[Direct link to the file & lines in question](https://github.com/ocornut/imgui/blob/321b84f01fbf3f64db6dbb53deb707d150ce30dc/imgui.h#L485)

**Comment by slynch8** [comment]
Yes, this is probably because of the macro, should start working when the preprocessor is in.
